### PR TITLE
Issue/89

### PR DESCRIPTION
**Pull request by RomanVolkov**
PR for https://github.com/SwiftWeekly/swiftweekly.github.io/issues/314
@SwiftWeekly/writers feel free to add your notes

### CODE REVIEW

File: _drafts/2017-09-28-issue-89.md [hunk 1]
```diff
@@ -1,58 +1,78 @@
...
+
+Swift core team gave [answer](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20170925/039907.html) about [Discussion: Resources](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20170918/039874.html) this week.
+> ....
+> We're currently defining our roadmap for SwiftPM in Swift 5, and it's not clear yet whether resource support will fit into this year or not, but either way it's clearly something we'll want to do. We're looking to encourage a thriving developer community around the Swift package manager, and we know that this will be important for many uses. If there are other limitations that are currently holding people back, we'd welcome feedback (and contributions!).
+> ....
 
 ### Finally
 
-> TODO: something funny/fun. tweet, link, etc.
+The problem [behind](https://twitter.com/xenadu02/status/911463433521860609) XCode simulator very slow performance is surprising.
```
Comment (jpsim) [inline _drafts/2017-09-28-issue-89.md]: "XCode" -> "Xcode".

Also, I'd move this into the "News and community" section since it's actual news, not just a humorous snippet that we usually feature in the "Finally" section.

Instead, I'd have this comment from apple/swift#11939 in the "Finally" section:

> tl;dr – Slicing a CoW should result in fillet mignon not ground chuck.

**Comment by jessesquires** [comment]
Thanks @RomanVolkov ! 🎉 Awesome job 👍

File: _drafts/2017-09-28-issue-89.md (hunk 1 above)
Comment (RomanVolkov) [reply _drafts/2017-09-28-issue-89.md]: I can't decided which section to use:) first was news, but I'm not sure that this news belongs to Swift ecosystem itself.
